product_margin_classification/views/view_product_product.xml

```diff
@@ -129,4 +129,33 @@ License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
         </field>
     </record>
 
+    <record id="action_apply_theoretical_price" model="ir.actions.server">
+        <field name="name">Apply Theoretical Price</field>
+        <field name="groups_id" eval="[(4, ref('base.group_user'))]"/>
```

Comment:
Here and elsewhere, I am not certain this is the correct value for groups_id.

Reply:
the function will change the sale price for many products. 
it should be available for people that can do that operation for a product.

Reply:
Changed this to `base.group_system`, which appears to be the group required for writing to product.product.

Reply:
That's a problem. base.group_system is erp manager. The feature will be hidden to all the users. To fix it : 
- set ``base.group_user``

(Core code : 
```access_product_product_employee,product.product employee,model_product_product,base.group_user,1,0,0,0```)

so the feature will be available for all the user that can see products.
Then an  error will be raised by ORM, if user can not write on products.

Reply:
reverted, fixed :+1: